packages/TokenRunner/src/Transformer/FixerTransformer/LineLengthTransformer.php

```diff
@@ -128,8 +128,13 @@ private function getFirstLineLength(int $startPosition, Tokens $tokens): int
         $currentPosition = $startPosition;
 
         while (! $this->isNewLineOrOpenTag($tokens, $currentPosition)) {
-            $lineLength += strlen($tokens[$currentPosition]->getContent());
+            $explode = explode("\n", $tokens[$currentPosition]->getContent());
+            $lineLength += strlen(end($explode));
```

Comment:
What does this do?

Requires comment or better new method

Reply:
what do you mean? `end` function? returns last element of array

Reply:
There last 2 lines